test/README.md

```diff
@@ -14,7 +14,7 @@ Our integration tests are based on functions from the samples project. To run in
    ```
    npm install -g azurite
    ```
-3. A local SQL Server instance - This is used by tests to verify that data is correctly added/fetched from the database when a test Function is run. You just need the server to be up and running, the tests will create the database and tables which will be cleaned up afterwards.
+3. A local SQL Server instance or an Azure SQL Database - This is used by tests to verify that data is correctly added/fetched from the database when a test Function is run. You just need the server to be up and running, the tests will create the database and tables which will be cleaned up afterwards.
 
    - You can either have a SQL Server installation with `localhost` available for connection via integrated security, or
```

Comment:
Instead of having these be bullet points let's make these sub-sections themselves with titles `Local Install`, `Docker Container`, and `Azure SQL Database`